spec/symbol_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'stock_quote/symbol'
+require 'stock_quote/utility'
+require 'spec_helper'
+
+describe StockQuote::Symbol  do
+  describe 'symbol_lookup' do
+    context 'success' do
+      describe 'apple company', vcr: { cassette_name: 'apple_lookup'} do
+        @fields = StockQuote::Symbol::FIELDS
+
+        @fields.each do |field|
+          it ".#{to_underscore(field)}" do
+            symbol = StockQuote::Symbol.symbol_lookup('apple').first
+            expect(symbol).to respond_to(to_underscore(field).to_sym)
+          end
+
+          it ".#{field}" do
+            symbol = StockQuote::Symbol.symbol_lookup('apple').first
+            expect(symbol).to respond_to(field.to_sym)
+          end
+        end
+
+        it 'returns sucessful result for query with company name' do
+          symbols = StockQuote::Symbol.symbol_lookup('apple')
+          expect(symbols.count).to eq(10)
+          expect(symbols.map(&:exch).uniq.sort).to eq(["EBS", "FRA", "GER", "LSE", "NAS", "NYQ", "PNK", "SHH"])
```

Comment:
yep, make sense. I didn't sort my array on the left side, but it does not help if new exchanges will be added later... 
so, changed it with https://github.com/tyrauber/stock_quote/pull/37/commits/e9c04865d0b8b7593400d75d858fe34b950d9c54